src/Munchkin.Core.Cards/Effects/GiveExtraTreasureRewardEffect.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Munchkin.Core.Contracts;
+using Munchkin.Core.Model;
+using Munchkin.Core.Model.Properties;
+
+namespace Munchkin.Core.Cards.Effects
+{
+    public class GiveExtraTreasureRewardEffect : IEffect<Table>
```

Comment:
This is not an issue, but maybe it makes sense to name it similar to the name of the attribute? Something like "TreasureBonusEffect", like there is "MonsterStrengthBonusEffect". I think it might help when searching for effects.